pvHelpers/certificate_bundle.py

```diff
@@ -0,0 +1,114 @@
+import codecs
+import os
+import re
+import ssl
+import subprocess
+import sys
+
+import certifi
+
+from .misc import g_log
+
+
+class CertificateBundle(object):
+    path = None
```

Comment:
why needs this?

Reply:
let make this `path` a required variable and remove the `set_path`.

this will avoid a lot of complications.